File: FizzBuzz.lua
function FizzBuzz(n)
	if n == 0 then
		return
	else
		FizzBuzz(n - 1)
		if n % 15 == 0 then
			print("FizzBuzz")
		elseif n % 3 == 0 then
			print("Fizz")
		elseif n % 5 == 0 then
			print("Buzz")
		else 
			print(n)
		end
	end
end

FizzBuzz(100)

File: FizzBuzz.java
class FizzBuzz {
	public static void main(String args[]) {
		FizzBuzz(100);
	}
   
   public static void FizzBuzz(int i) {
      if(i==0) return;
      else {
         FizzBuzz(i-1);
         if(i%3==0&&i%5==0) System.out.println("FizzBuzz");
			else if(i%3==0) System.out.println("Fizz");
			else if(i%5==0) System.out.println("Buzz");
			else System.out.println(i);
      }
   }
}